### ISSUE TITLE
Add detailed setup instructions

### ISSUE THREAD

**Issue by karras** [adfinis/gitlab-pkgbot#3]
We should add some more detailed setup instructions, especially briefly describe how the bot works in case someone is not familiar with aptly.

**Comment by brejoc** [comment]
I assume `aptly` also needs to be installed besides `python-yaml` and `python-pip`. But how about `pyaptly`? Is that also needed or are the relevant parts integrated into `gitlab-pkgbot`?

**Comment by karras** [comment]
Yes, currently pyaptly (see https://github.com/adfinis-sygroup/pyaptly) is required as well but we want to move away from it. Mainly because in most cases you'll directly publish the aptly repo without any snapshots. This also makes the process faster as publishing with aptly usually takes some minutes.

Actually running the commands against the aptly API would be even nicer but on our own aptly server we manage multiple repos and make use of pyaptly for that. Sadly it's currently not possible to run aptly as an API and through the CLI at the same time.
